packages/reporter-vue/src/runnables/Runnable.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <div :class="`runnable-root ${state}-state`">
+    <BaseAccordion v-model="open">
```

Comment:
Completely optional but we could consider the new `v-model` modifier syntax for more clarity: `<BaseAccordion v-model:open="open">`. Not necessary but good to keep in mind.